src/server/game/Maps/MapInstanced.cpp

```diff
@@ -278,8 +278,8 @@ bool MapInstanced::DestroyInstance(InstancedMaps::iterator &itr)
     return true;
 }
 
-bool MapInstanced::CanEnter(Player* /*player*/)
+Map::EnterState MapInstanced::CannotEnter(Player* player)
```

Comment:
home/travis/build/TrinityCore/TrinityCore/src/server/game/Maps/MapInstanced.cpp:281:51: fatal error: 
      unused parameter 'player'
      [-Wunused-parameter]
  ...player)
     ^

Just comment it out again, even though I do not know why it needs player as handed variable.